share/picongpu/tests/Pusher/include/picongpu/param/species.param

```diff
@@ -0,0 +1,103 @@
+     *  - particles::shapes::TSC : Assignment function is a piecewise quadratic spline
+     *  - particles::shapes::PQS : Assignment function is a piecewise cubic spline
+     *  - particles::shapes::PCS : Assignment function is a piecewise quartic spline
+     */
+    using UsedParticleShape = particles::shapes::CIC;
+
+    /** select interpolation method to be used for interpolation of grid-based field values to particle positions
+     */
+    using UsedField2Particle = FieldToParticleInterpolation<UsedParticleShape, AssignedTrilinearInterpolation>;
+
+    /*! select current solver method
+     * - currentSolver::Esirkepov< SHAPE, STRATEGY > : particle shapes - CIC, TSC, PQS, PCS (1st to 4th order)
+     * - currentSolver::EmZ< SHAPE, STRATEGY >       : particle shapes - CIC, TSC, PQS, PCS (1st to 4th order)
+     * - currentSolver::EZ< SHAPE, STRATEGY >        : same as EmZ (just an alias to match the currently used naming)
+     *
+     * STRATEGY (optional):
+     * - currentSolver::strategy::StridedCachedSupercells
+     * - currentSolver::strategy::StridedCachedSupercellsScaled<N> with N >= 1
+     * - currentSolver::strategy::CachedSupercells
+     * - currentSolver::strategy::CachedSupercellsScaled<N> with N >= 1
+     * - currentSolver::strategy::NonCachedSupercells
+     * - currentSolver::strategy::NonCachedSupercellsScaled<N> with N >= 1
+     */
+    using UsedParticleCurrentSolver = currentSolver::Esirkepov<UsedParticleShape>;
+
+/** particle pusher configuration
+ *
+ * Defining a pusher is optional for particles
+ *
+ * - particles::pusher::HigueraCary : Higuera & Cary's relativistic pusher preserving both volume and ExB velocity
+ * - particles::pusher::Vay : Vay's relativistic pusher preserving ExB velocity
+ * - particles::pusher::Boris : Boris' relativistic pusher preserving volume
+ * - particles::pusher::ReducedLandauLifshitz : 4th order RungeKutta pusher
+ *                                              with classical radiation reaction
+ * - particles::pusher::Composite : composite of two given pushers,
+ *                                  switches between using one (or none) of those
+ *
+ * For diagnostics & modeling: ------------------------------------------------
+ * - particles::pusher::Acceleration : Accelerate particles by applying a constant electric field
+ * - particles::pusher::Free : free propagation, ignore fields
+ *                             (= free stream model)
+ * - particles::pusher::Photon : propagate with c in direction of normalized mom.
+ * - particles::pusher::Probe : Probe particles that interpolate E & B
+ * For development purposes: --------------------------------------------------
+ * - particles::pusher::Axel : a pusher developed at HZDR during 2011 (testing)
+ */
+#ifndef PARAM_PARTICLEPUSHER
+#    define PARAM_PARTICLEPUSHER Boris
+#endif
+    using UsedParticlePusher = particles::pusher::PARAM_PARTICLEPUSHER;
```

Comment:
could we remove those param files that do not add something compared to the default?